rfc/draft-oberstet-hybi-tavendo-wamp-00.md

```diff
@@ -41,91 +41,84 @@ This document defines the Web Application Messaging Protocol (WAMP). WAMP is a r
 
 _This section is non-normative._
 
-The WebSocket protocol brought bi-directional real-time connections to the browser. This defines an API at the message level, requiring users who want to use WebSocket connections in their applications to define any semantics on top of it.
+The WebSocket protocol brings bi-directional real-time connections to the browser. It defines an API at the message level, requiring users who want to use WebSocket connections in their applications to define their own semantics on top of it.
 
 The Web Application Messaging Protocl (WAMP) was initially defined as a WebSocket sub-protocol at the technical level, and is intended to provide application developers with the semantics they need to handle messaging between components in distributed applications.
 
-WAMP is a routed protocol, with all components connecting to a WAMP router.
+WAMP is a routed protocol, with all components connecting to a _WAMP router_.
 
-WAMP provides two messaging patterns: Publish & Subscribe and routed Remote Procedure calls.
+WAMP provides two messaging patterns: _Publish & Subscribe_ and _routed Remote Procedure Calls_.
 
-Publish & Subscribe is an established messaging pattern where components inform the router that they want to receive information on a topic (they suscribe to the topic). A component can then publish to this topic, and the router distributes events to all subscribers.
+Publish & Subscribe (PubSub) is an established messaging pattern where a component, the _Subscriber_, informs the router that it wants to receive information on a topic (i.e., it subscribes to a topic). Another component, a _Publisher_, can then publish to this topic, and the router distributes events to all Subscribers.
 
-With routed Remote Procedure calls, the decoupling of the Publish & Subscribe pattern is applied to Remote Procedure Calls: a component announces to the router that it provides a certain procedure, identified by a procedure name. Other components can then call the procedure, with the router invoking the procedure on the registering component, receiving the result for this, and forwarding this to the caller.
+Routed Remote Procedure Calls (RPCs) rely on the same sort of decoupling that is used by the Publish & Subscribe pattern. A component, the _Callee_, announces to the router that it provides a certain procedure, identified by a procedure name. Other components, _Callers_, can then call the procedure, with the router invoking the procedure on the Callee, receiving the procedure's result, and then forwarding this result back to the Caller. Routed RPCs differ from traditional client-server RPCs in that the router serves as an intermediary between the Caller and the Callee.
```

Comment:
The term "Callee" was used in this subsection without having been defines. I decided to restructure his paragraph so that it gives the reader a general idea of what "Caller" and "Callee" really means.

I know that personally, I was not familiar with the term "Callee" before I had started using WAMP.